src/hooks/useBuyout/index.ts

```diff
@@ -58,7 +58,7 @@ function handleBuyoutError(error: string) {
 function generateBuyoutExtrinsicPayload(amount: BigNumber) {
   const amountString = amount.toString();
   // We always go the 'buyout' route because the 'exchange' route does not work sometimes
-  return { buyout: { amount: amountString } };
+  return { Buyout: { amount: amountString } };
```

Comment:
This capitalization change is necessary because the updated types now require a type similar to 
```
{ Buyout : any } | { Exchange : any } 
```
and the linting complains if the capitalization doesn't match.